telegram/ext/_applicationbuilder.py

```diff
@@ -1016,6 +1019,42 @@ async def post_shutdown(application: Application) -> None:
         self._post_shutdown = post_shutdown
         return self
 
+    def post_stop(
+        self: BuilderType, post_stop: Callable[[Application], Coroutine[Any, Any, None]]
+    ) -> BuilderType:
+        """
+        Sets a callback to be executed by :meth:`Application.run_polling` and
+        :meth:`Application.run_webhook` *after* executing :meth:`Updater.stop`
+        and :meth:`Application.stop`.
+
```

Comment:
missing versionadded

Reply:
The versionadded directive isn't in the above methods that are introduced in v20, i.e. `post_init` and `post_shutdown`. Shouls I add it in those also?

Reply:
no, Application is a new class in v20 itself and those methods came with it, but this one will only be visible from 20.1 onwards